src/components/details.jsx/styles.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+export const Info = styled.p`
+  font-size: 1.4rem;
+`;
+
+export const Title = styled.h1`
+  margin-top: 1rem;
+`;
```

Comment:
- [ ] Please follow the design for the details page as well. Color is fine, but try to keep the same `Layout: composition and space between elements`.

### Given design
![Screenshot from 2021-09-19 00-05-26](https://user-images.githubusercontent.com/22434660/133905100-66acd681-73a1-402f-9b89-674be0511f7e.png)

### Your design

![Screenshot from 2021-09-18 23-54-00](https://user-images.githubusercontent.com/22434660/133905124-dbef0e05-34f9-4f35-91f4-fb7a40d92756.png)

Reply:
You have made the requested change by the previous reviewer and that is good. But I suggest you should also stick with the original background color as it is a design requirement. :pray: